tests/bootstrap.php

```diff
@@ -2,4 +2,18 @@
 
 declare(strict_types=1);
 
-require __DIR__ . '/../vendor/autoload.php';
+$_GET['site'] = 'default';
+$ignoreAuth = true;
+require_once(__DIR__ . "/../interface/globals.php");
```

Comment:
to provide a bit of context on this, the above script is OpenEMR's god script (is basically run by every script) that keeps everything together and brings in autoloader and ensures authentication (the `$ignoreAuth = true;` flag skips authentication).
That's why below code isn't needed.